src/main/java/org/broadinstitute/consent/http/db/DacDAO.java

```diff
@@ -39,17 +39,53 @@ public interface DacDAO extends Transactional<DacDAO> {
    *
    * @return List<Dac>
    */
+  @RegisterBeanMapper(value = DataAccessAgreement.class, prefix = "daa")
+  @RegisterBeanMapper(value = FileStorageObjectDAO.class)
   @RegisterBeanMapper(value = Dac.class)
   @RegisterBeanMapper(value = Dataset.class)
   @UseRowReducer(DacReducer.class)
   @SqlQuery("""
-      SELECT dac.dac_id, dac.email, dac.name, dac.description, d.dataset_id, d.name AS dataset_name,
-        DATE(d.create_date) AS dataset_create_date, d.object_id, d.active, d.needs_approval,
-        d.alias AS dataset_alias, d.create_user_id, d.update_date AS dataset_update_date,
-        d.update_user_id, d.data_use AS dataset_data_use, d.sharing_plan_document,
-        d.sharing_plan_document_name
+      SELECT
+        dac.dac_id,
+        dac.email,
+        dac.name,
+        dac.description,
+        d.dataset_id,
+        d.name AS dataset_name,
+        DATE(d.create_date) AS dataset_create_date,
+        d.object_id,
+        d.active,
+        d.needs_approval,
+        d.alias AS dataset_alias,
+        d.create_user_id,
+        d.update_date AS dataset_update_date,
+        d.update_user_id,
+        d.data_use AS dataset_data_use,
+        d.sharing_plan_document,
+        d.sharing_plan_document_name,
+        daa.daa_id AS daa_daa_id,
+        daa.create_user_id AS daa_create_user_id,
+        daa.create_date AS daa_create_date,
+        daa.update_user_id AS daa_update_user_id,
+        daa.update_date AS daa_update_date,
+        daa.initial_dac_id AS daa_initial_dac_id,
+        fso.file_storage_object_id AS fso_file_storage_object_id,
+        fso.entity_id AS fso_entity_id,
+        fso.file_name AS fso_file_name,
+        fso.category AS fso_category,
+        fso.gcs_file_uri AS fso_gcs_file_uri,
+        fso.media_type AS fso_media_type,
+        fso.deleted AS fso_deleted,
+        fso.delete_user_id AS fso_delete_user_id,
+        fso.create_date AS fso_create_date,
+        fso.create_user_id AS fso_create_user_id,
+        fso.update_date AS fso_update_date,
+        fso.update_user_id AS fso_update_user_id
       FROM dac
       LEFT OUTER JOIN dataset d ON dac.dac_id = d.dac_id
+      LEFT JOIN dac_daa dd ON dac.dac_id = dd.dac_id
```

Comment:
I think that `LEFT OUTER JOIN` and `LEFT JOIN` are equivalent -- does this repo favor one representation over the over?  Can we make the joins consistent within this query?